spring-annotation/build.gradle

```diff
@@ -7,20 +7,20 @@ dependencies {
     api mn.micronaut.inject
     api mn.micronaut.aop
     api mnValidation.micronaut.validation
-    implementation mnValidation.micronaut.validation.processor
 
+    implementation mnValidation.micronaut.validation.processor
     implementation mn.micronaut.core.processor
     implementation projects.micronautSpring
     implementation projects.micronautSpringContext
-
     implementation mn.micronaut.runtime
-    implementation(mnCache.micronaut.cache.core)
```

Comment:
I agree that you can write parentheses everywhere. But I just brought the file to one view, because all the dependencies were without brackets, except this one.